docs/en/build-store/refresh-control.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Validations and refresh control
+table_of_contents: true
+---
+
+# Validations and refresh control
+
+## What is refresh control?
+
+_“🎵 baby don't update me, don't update me no more 🎵”_
+
+
+Refresh control is a mechanism for controlling the revisions of a snap that will be offered as updates  to a device. A *gating snap* can control the revisions of a *gated snap* so that, if the *gating snap* is installed on a device, only revisions of the *gated snap* which have been validated by the publisher of the *gating snap* can be installed or refreshed to.
```

Comment:
User docs should not use the future tense "will" unless it really means the future and it should be avoided. This could be rephrased as "Refresh Control holds target snaps at specified revisions."

Reply:
I'll leave this for a style correction pass once it's merged.

Reply:
Who is going to do these style corrections? I propose thatthis is the time, and you are the author, so go for it!